python/mxnet/ndarray/numpy/random.py

```diff
@@ -17,5 +17,65 @@
 
 """Namespace for operators used in Gluon dispatched by F=ndarray."""
 from __future__ import absolute_import
+from ...context import current_context
+from . import _internal as _npi
 
-__all__ = []
+__all__ = ['uniform']
+
+
+def uniform(low=0.0, high=1.0, size=None, ctx=None, dtype=None, out=None):
```

Comment:
Argument order here should be the same as your documentation.

Reply:
> Argument order here should be the same as your documentation.

fixed! thx for suggestion.